packs/feats/rain-of-bolts.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "ancestry",
         "description": {
-            "value": "<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p>Throughout the day, your body produces powerful projectiles within your chassis. You fire them all at once in either a @Template[cone|distance:30]{30-foot cone} or a @Template[emanation|distance:10]{10-foot emanation}. Foes in the area take @Damage[(3 + floor(@actor.level/3))d6[piercing]] damage with a @Check[reflex|against:class-spell|basic] against the higher of your class DC or spell DC. This increases to 7d6 at 12th level, 8d6 at 15th level, and 9d6 at 18th level.</p>\n<p><strong>Enhancement</strong> The projectiles regenerate quicker, they are much more powerful, and you can refine the metallic content to harm certain creatures. You can use Rain of Bolts once per hour instead of once per day. The damage increases to @Damage[12d6[piercing]]{12d6} and each time you use the action, you can choose adamantine, cold iron, or silver. The damage from Rain of Bolts is treated as the metal you chose. At 20th level, the damage increases to @Damage[13d6[piercing]]{13d6}.</p>"
+            "value": "<p><strong>Frequency</strong> once per day</p><hr /><p>Throughout the day, your body produces powerful projectiles within your chassis. You fire them all at once in either a @Template[cone|distance:30]{30-foot cone} or a @Template[emanation|distance:10]{10-foot emanation}. Foes in the area take @Damage[(3 + floor(@actor.level/3))d6[piercing]|options:area-damage] damage with a @Check[reflex|against:class-spell|basic|options:area-effect] against the higher of your class DC or spell DC. This increases to 7d6 at 12th level, 8d6 at 15th level, and 9d6 at 18th level.</p>\n<p><strong>Enhancement</strong> The projectiles regenerate quicker, they are much more powerful, and you can refine the metallic content to harm certain creatures. You can use Rain of Bolts once per hour instead of once per day. The damage increases to @Damage[12d6[piercing]|options:area-damage|shortLabel] and each time you use the action, you can choose adamantine, cold iron, or silver. The damage from Rain of Bolts is treated as the metal you chose. At 20th level, the damage increases to @Damage[13d6[piercing]|options:area-damage|shortLabel].</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Frequency</strong> once per day</p><hr /><p>Throughout the day, your body produces powerful projectiles within your chassis. You fire them all at once in either a @Template[cone|distance:30] or a @Template[emanation|distance:10]. Foes in the area take @Damage[(3 + floor(@actor.level/3))d6[piercing]|options:area-damage] damage with a @Check[reflex|against:class-spell|basic|options:area-effect] against the higher of your class DC or spell DC. This increases to 7d6 at 12th level, 8d6 at 15th level, and 9d6 at 18th level.</p>\n<p><strong>Enhancement</strong> The projectiles regenerate quicker, they are much more powerful, and you can refine the metallic content to harm certain creatures. You can use Rain of Bolts once per hour instead of once per day. The damage increases to @Damage[12d6[piercing]|options:area-damage|shortLabel] and each time you use the action, you can choose adamantine, cold iron, or silver. The damage from Rain of Bolts is treated as the metal you chose. At 20th level, the damage increases to @Damage[13d6[piercing]|options:area-damage|shortLabel].</p>"
```